packages/api/src/controllers/organizations/v2/mod.rs

```diff
@@ -0,0 +1,45 @@
+use by_axum::{
+    auth::Authorization,
+    axum::{
+        extract::{Query, State},
+        routing::get,
+        Extension, Json,
+    },
+};
+use models::*;
+
+#[derive(Clone, Debug)]
+pub struct OrganizationControllerV2 {
+    repo: OrganizationRepository,
+}
+
+impl OrganizationControllerV2 {
+    pub fn route(pool: sqlx::Pool<sqlx::Postgres>) -> Result<by_axum::axum::Router> {
+        let repo = Organization::get_repository(pool.clone());
+
+        let ctrl = OrganizationControllerV2 { repo };
+
+        Ok(by_axum::axum::Router::new()
+            .route("/", get(Self::list_organization))
+            .with_state(ctrl.clone()))
+    }
+
+    pub async fn list_organization(
+        State(ctrl): State<OrganizationControllerV2>,
+        Extension(_auth): Extension<Option<Authorization>>,
+        Query(param): Query<OrganizationParam>,
+    ) -> Result<Json<OrganizationGetResponse>> {
+        tracing::debug!("list_organization {:?}", param);
+
+        match param {
+            OrganizationParam::Query(q) => {
+                let organizations: QueryResponse<OrganizationSummary> = ctrl.repo.find(&q).await?;
+                Ok(Json(OrganizationGetResponse::Query(organizations)))
+            }
+            OrganizationParam::Read(_action) => {
+                // FIXME: return filtered organization
```

Comment:
```suggestion
                // TODO: implement to return filtered organization
```
FIXME 는 현재도 정상 동작하지만 개선하면 좋은걸 의미하는데, 지금은 구현 자체가 안됐기 때문에 TODO 를 쓰는게 맞습니다.

Reply:
저도 따로 계획은 없는데 OrganizationParam::Read 조건도 처리해야한다고 해서 혹시몰라 넣었습니다